.github/workflows/nuget-package-tests.yml

```diff
@@ -15,35 +15,36 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-latest, ubuntu-latest, macos-latest]
+        os: [windows-latest, ubuntu-20.04, macos-latest]
```

Comment:
nit: be consistent with version there (everywhere pinning or everywhere latest - I personally prefer the first one)

Reply:
```suggestion
        os: [windows-latest, ubuntu-latest, macos-latest]
```